amplipi/app.py

```diff
@@ -229,6 +230,23 @@ def shutdown():
   Popen('sleep 1 && sudo systemctl poweroff', shell=True)
 
 
+@api.post(
+    '/api/persist_logs',
+    tags=["config"],
+    response_class=Response,
+    responses={
+      200: {}
+    }
+  )
+def log_persistance(ctrl: Api = Depends(get_ctrl)):
+  """Toggles log persistance by directly changing the journald.conf and restarting the service"""
+  try:
+    ctrl.toggle_persist_logs()
+  except Exception as e:
+    logger.error("persist_logs toggle failed!")
+    raise e
```

Comment:
I found reasons to call this function from within the Api object itself and so moved it there, this imports the Api object as ctrl and calls the function